prow/config/config.yaml

```diff
@@ -565,7 +565,8 @@ branch-protection:
               required_status_checks:
                 contexts:
                   - "jenkins-docs/verify"
-                  - "pull"
+                  - "tidb-check"
+                  - "tidb-cloud-check"
```

Comment:
Which PR will change this workflow in `master` branch?

Reply:
yes,  https://github.com/pingcap/docs/pull/9354/files